### PR TITLE
Coordinator improvements

### PR DESCRIPTION
**Pull request by nakul2234**
Consists of tweaks requested by coordinators to help the interface of Osler.

### CODE REVIEW

File: pttrack/forms.py [hunk 1]
```diff
@@ -18,38 +18,56 @@ class Meta:
...
+                                                        required = False,
+                                                        help_text=PATIENT_REACHED_HELP, 
+                                                        label="Were you able to contact the patient?")
+
     class Meta:
         model = followup_models.ReferralFollowup
-        exclude = ['patient', 'author', 'author_type']
+        fields = ['contact_method',
+            'contact_resolution',
+            'patient_reached',
```
Comment (justinrporter) [inline pttrack/forms.py]: I was thinking that we would add a boolean property to the contact resolution--if it was a wrong number or whatever we *know* whether the person was successful.

File: pttrack/followup_models.py [hunk 2]
```diff
@@ -141,9 +143,11 @@ def short_text(self):
 class ReferralFollowup(Followup):
     '''Datamodel for a PCP referral followup.'''
 
+    # Template relies on following variable to render Admin Edit. If you change the variable here, you must edit patient_detail.html
     REFTYPE_HELP = "What kind of provider was the patient referred to?"
     referral_type = models.ForeignKey(mymodels.ReferralType,
-                                      help_text=REFTYPE_HELP)
+                                      help_text=REFTYPE_HELP, 
+                                      default="Other")
```
Comment (justinrporter) [inline pttrack/followup_models.py]: I don't love this--it's going to encourage people to put other when they're in a rush... I can't recall why we decided to do this...

**Comment by justinrporter** [comment]
Also, could we get a test for the validation on that form? Break it out into a test_forms.py file (django will automatically detect it and run tests against it if a file name starts with test_*).

File: pttrack/followup_models.py (hunk 2 above)
Comment (nakul2234) [inline pttrack/followup_models.py]: Its just so it won't be required, when they did not contact the patient anyways.

Comment (justinrporter) [inline pttrack/followup_models.py]: Why are we not requiring this? @radhikajain 

Referral to the COH can mean a lot of things, for example...

Comment (nakul2234) [inline pttrack/followup_models.py]: Okay works for me

File: pttrack/forms.py (hunk 1 above)
Comment (nakul2234) [inline pttrack/forms.py]: Makes sense, will edit it

**Comment by justinrporter** [comment]
Also heads up, I created merge conflicts in this branch by fixing the bug in the home screen view. Turned out to be an indentation error. No idea how that snuck in there.

**Comment by nakul2234** [comment]
@justinrporter This should be good to go. Also, since Anatomy is coming up ill probably be less active this next week.

**Comment by nakul2234** [comment]
@justinrporter I also added in the phone directory that they requested. Let me know when you are cool with pushing and updating the system. I wanted to send a changelog email so the coordinators know what to expect.

**Pull request merged**